tests/test_gitlab.py

```diff
@@ -54,7 +58,7 @@ class TestGitlabIssue(AbstractServiceTest, ServiceTest):
             "id": 1,
             "title": "v1.0",
             "description": "",
-            "due_date": "2012-07-20",
+            "due_date": arbitrary_duedate.date().isoformat(),
```

Comment:
I think the `date()` part is the main problem with the tests. It converts type from `datetime` to `date` which causes it to lose it's timezone awareness.